azure/packages/azure-client/src/test/AzureClient.spec.ts

```diff
@@ -164,4 +165,32 @@ describe("AzureClient", () => {
         // eslint-disable-next-line require-atomic-updates
         console.error = consoleErrorFn;
     });
+
+    /**
+     * Scenario: Test if AzureClient starts with the container in `write` mode.
+     *
+     * Expected behavior: AzureClient creates a connected container in `write` mode.
+     */
+    it("creates a container in write mode", async () => {
+        const { container } = await client.createContainer(schema);
+        await container.attach();
+
+        const map1 = container.initialObjects.map1 as SharedMap;
+        map1.set("newpair-id", "test");
+
+        await timeoutPromise((resolve) => container.once("connected", resolve), {
+            durationMs: 1000,
+            errorMsg: "container connect() timeout",
+        });
+
+        // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-explicit-any
+        const forcedWriteConnectionMode = (container as any).container
+            .connectionMode as ConnectionMode;
+
+        assert.strictEqual(
```

Comment:
Can we test both load and create scenarios? Can we also test against both read and write permissions?